packages/web/package.json

```diff
@@ -1,54 +1,46 @@
 {
-  "name": "web-app",
+  "name": "@entria/web",
   "version": "1.0.0",
+  "license": "MIT",
+  "private": true,
   "main": "index.js",
-  "engines": {
-    "node": ">= 10.11.0"
-  },
   "scripts": {
-    "start": "cross-env NODE_ENV=development wp --config ./webpack.config",
-    "build": "cross-env NODE_ENV=production wp --config ./webpack.config",
+    "start": "cross-env NODE_ENV=development wp --config ./webpack/webpack.dev",
+    "build": "cross-env NODE_ENV=production wp --config ./webpack/webpack.prod",
     "jest:lint": "jest --config jest-eslint.config.js",
     "test": "jest --coverage",
-    "ts:check": "tsc"
+    "ts:check": "tsc --noEmit"
```

Comment:
[nit] I think that since `web/tsconfig.json` file extends the root `tsconfig.json` and the `noEmit` options is set there, there is no need to add it here 🤔